atlasdb-client/src/main/java/com/palantir/atlasdb/table/description/render/StreamStoreRenderer.java

```diff
@@ -849,6 +886,8 @@ private void cellsCleanedUp() {
         Generated.class,
         LZ4CompressingInputStream.class,
         LZ4BlockInputStream.class,
+        GzipCompressingInputStream.class,
+        GZIPInputStream.class,
```

Comment:
this.. probably does not contain enough of the inputs you need. I think you should be able to remove the imports for the LZ4 and GZIP streams, and instead just have a single import for your enum.